### ISSUE TITLE
A11y_Oracle Assessment for Azure Data Studio (ADS)_AssessmentReport_SkuRecommendations_Keyboard:Focus order for tab items is incorrect at "SKU Recommendations"

### ISSUE THREAD

**Issue by v-akr** [microsoft/azuredatastudio#19580]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
 #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win11;#DesktopApp;#WCAG2.4.3;#AINotInScope;#FTP;#Keyboard;

### Environment Details:
Application Name: Oracle Assessment for Azure Data Studio (ADS)
Operating System: Windows 11
Version :1.35.1

### Steps to Reproduce:

1. Download the azure data studio.
2. Open the application and tab to the Connections icon in left navigation.
3. Tab to the new connections button and press enter. Connection dialog box will open.
4. Fill Correct details in connection details and tab to the connect button and press enter. New connection will be added.
5. Tab to the Database Migration Assessment for Oracle button and then press enter. "Database Migration Assessment for Oracle" page will be opened.
6. Tab to the "Run New Assessment" button at top the page and press enter.New Database Migration Assessment for Oracle dialog box will open. Fill the correct details. Tab to "Start Assessment button and press enter.
7. New Assessment can be seen in last five assessment tab. Tab to the assessment press enter. New tab will get open.
8. Tab to Assessment overview by platforms section. Tab to Assessment for Azure virtual machine in the section.Tab to view report details button and press enter. New screen will open.
9. Tab to "Sku Recommendations" tab and press enter.
10. Navigate using keyboard and observe the tab items.

### Actual:
While navigating with keyboard after selecting the "SKU Recommendations" tab item , Keyboard focus is not shifting to the data of selected tab item instead its moving to the "Feature compatability"  tab item. 

### Expected:
While navigating with keyboard after selecting the "SKU Recommendations" tab item , Next Tab focus should be shifting to the data of selected tab item.

### User Impact:
Keyboard users will face difficulty to navigate in the selected tab item.
